clj/build.boot

```diff
@@ -51,6 +62,16 @@
            (install)
            (target)))
 
+(deftask build-cljs
+  "build clojurescript"
+  []
+  (comp
+   (test-cljs)
+   (cljs :optimizations :advanced)))
+
+(deftask build []
+  (comp (build-cljs) (build-clj)))
```

Comment:
It seems to be a result of setting `:exit?` to true.  It seems to exit whether or not there are errors.  Both are built if `:exit?` is false.